src/dotnet/APIView/APIViewWeb/Languages/CppLanguageService.cs

```diff
@@ -121,7 +121,7 @@ public override async Task<CodeFile> GetCodeFileAsync(string originalName, Strea
             astStream.Position = 0;
 
             //Generate pacakge name from original file name
-            var packageNamespace = originalName.Substring(0,originalName.LastIndexOf(".")).Replace("_", "::");
+            var packageNamespace = originalName.Substring(0,originalName.LastIndexOf(".")).Replace("_", "::").ToLower();
```

Comment:
It would be better to have users select the package from a dropdown.  That way typos in the .ast filename won't result is extra reviews

Reply:
Reason for not adding a package dropdown at this moment is to avoid impact for other languages. We can surely look at this as an enhancement in the future.